schematic/utils/validate_utils.py

```diff
@@ -2,27 +2,27 @@
 from schematic.utils.io_utils import load_json
 from jsonschema import validate
 
-from definitions import DATA_PATH
+from schematic import CONFIG
 
 def validate_schema(schema):
     """Validate schema against schema.org standard
     """
-    json_schema_path = os.path.join(DATA_PATH, 'validation_schemas', 'schema.json')
+    json_schema_path = os.path.join('validation_schemas', 'schema.json')
```

Comment:
This implicitly relies on the working directory. This path should be made explicit, possibly through the config file, but I wonder if the user needs to perform this task. I think the `config.yml` file should be restricted to user-facing stuff, but that's just my opinion.